platform/backend_connectors/basic_sql_backend_connector.go

```diff
@@ -72,10 +74,12 @@ func (p *BasicSqlBackendConnector) Query(ctx context.Context, query string, args
 }
 
 func (p *BasicSqlBackendConnector) QueryRow(ctx context.Context, query string, args ...interface{}) quesma_api.Row {
+	logger.Info().Msgf("queryRow sql: %s", query)
 	return p.connection.QueryRowContext(ctx, query, args...)
 }
 
 func (p *BasicSqlBackendConnector) Exec(ctx context.Context, query string, args ...interface{}) error {
+	logger.Info().Msgf("Exec sql: %s", query)
```

Comment:
All three above make a huge sense for debugging. You may also leverage our internal debug console `http://localhost:9999`.

Reply:
thx